src/sass/components/UploadForm.scss

```diff
@@ -58,10 +58,9 @@
     }
 
     button {
```

Comment:
We have 2 `button` in this file, both within `.dropzone` ... see line 71.
Can we bring these together in this PR?